src/main.rs

```diff
@@ -366,20 +376,22 @@ fn main() -> io::Result<()> {
             SubCommand::with_name("compare")
                 .arg(
                     Arg::with_name("gfa1")
-                        .short("g1")
+                        .short("1")
```

Comment:
I changed the name of the arguments because short args can't have more than one letter